scripts/alpine-build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+echo "Building an alpine standalone binary..."
+cd snyk
+# install pkg globally so that the local deps do not get polluted with it
+# prevents non-deterministic tests, as they depend on the deps being used
+npm i -g pkg
+pkg . -t alpine -o snyk-alpine
```

Comment:
Please change `snyk-alpine` to `snyk-to-html-alpine`

Reply:
also, `-t alpine` better be `-t node8-alpine-x64` to make sure we get to right node runtime version.